src/main/java/lotto/Winner.java

```diff
@@ -0,0 +1,34 @@
+package lotto;
+
+import java.util.Arrays;
+
+public enum Winner {
+	FIRST_PRIZE(6, 2000000000L),
+	SECOND_PRIZE(5, 1500000L),
+	THIRD_PRIZE(4, 50000L),
+	FOURTH_PRIZE(3, 5000L),
```

Comment:
```suggestion
	FIRST_PRIZE(6, 2_000_000_000L),
	SECOND_PRIZE(5, 1_500_000L),
	THIRD_PRIZE(4, 50_000L),
	FOURTH_PRIZE(3, 5_000L),
```
이렇게 표현하면 가독성에 더 좋답니다!

Reply:
ㄴ😲ㄱ 엄청난 꿀팁 감사합니다.